### DISCUSSION TITLE
connecting encoder with multiplex

### DISCUSSION THREAD

**Discussion by Zlraul** [tttapa/Control-Surface#521] [Q&A]
hey, i need help with connecting encoder through a multiplex. i just wonder is  there any way to do this
or it can only be connected straight to the board?
i'm using a normal encoder without detent with a 74HC4067.

**Comment by tttapa** [comment]
Unfortunately this is not possible. The reasons behind this are explained in the FAQ: https://tttapa.github.io/Control-Surface-doc/Doxygen/da/dc1/FAQ.html#faq-mux-encoder

**Reply by Zlraul** [reply]
hmm okay, then what about connecting a keypad with a multiplex? bc if I can move the keypad through a multiplex I will have a room for the encoder

**Reply by tttapa** [reply]
> what about connecting a keypad with a multiplex?

Yes, but only the column pins. The row pins have to be outputs (which is not possible using the multiplexer class).
